test/js/sql/sqlite-sql.test.ts

```diff
@@ -1311,6 +1311,130 @@ describe("SQL helpers", () => {
     expect(results[0].value).toBe("test");
   });
 
+  test("insert select helper with IN", async () => {
+    const random_name = "test_" + randomUUIDv7("hex").replaceAll("-", "");
+    await sql`CREATE TEMPORARY TABLE ${sql(random_name)} (id int, name text, age int)`;
+    {
+      await sql`INSERT INTO ${sql(random_name)} ${sql({ id: 1, name: "John", age: 30 })}`;
+      const result = await sql`SELECT * FROM ${sql(random_name)}`;
+      expect(result[0].id).toBe(1);
+      expect(result[0].name).toBe("John");
+      expect(result[0].age).toBe(30);
+    }
+    await sql`CREATE TEMPORARY TABLE ${sql(random_name + "2")} (id int, name text, age int)`;
+    {
+      await sql`INSERT INTO ${sql(random_name + "2")} (id, name, age) SELECT id, name, age FROM ${sql(random_name)} WHERE id IN ${sql([1, 2])}`;
+      const result = await sql`SELECT * FROM ${sql(random_name + "2")}`;
+      expect(result[0].id).toBe(1);
+      expect(result[0].name).toBe("John");
+      expect(result[0].age).toBe(30);
+    }
+  });
+
+  test("update helper with undefined values", async () => {
+    const random_name = "test_" + randomUUIDv7("hex").replaceAll("-", "");
+    await sql`CREATE TEMPORARY TABLE ${sql(random_name)} (id int, name text, age int)`;
+    const users = [
+      { id: 1, name: "John", age: 30 },
+      { id: 2, name: "Jane", age: 25 },
+    ];
+    await sql`INSERT INTO ${sql(random_name)} ${sql(users)}`;
+
+    await sql`UPDATE ${sql(random_name)} SET ${sql({ name: "Mary", age: undefined })} WHERE id IN ${sql([1, 2])}`;
+    const result = await sql`SELECT * FROM ${sql(random_name)}`;
+    expect(result[0].id).toBe(1);
+    expect(result[0].name).toBe("Mary");
+    expect(result[0].age).toBe(30);
+    expect(result[1].id).toBe(2);
+    expect(result[1].name).toBe("Mary");
+    expect(result[1].age).toBe(25);
+  });
+  test("update helper that starts with undefined values", async () => {
+    const random_name = "test_" + randomUUIDv7("hex").replaceAll("-", "");
+    await sql`CREATE TEMPORARY TABLE ${sql(random_name)} (id int, name text, age int)`;
+    const users = [
+      { id: 1, name: "John", age: 30 },
+      { id: 2, name: "Jane", age: 25 },
+    ];
+    await sql`INSERT INTO ${sql(random_name)} ${sql(users)}`;
+
+    await sql`UPDATE ${sql(random_name)} SET ${sql({ name: undefined, age: 19 })} WHERE id IN ${sql([1, 2])}`;
+    const result = await sql`SELECT * FROM ${sql(random_name)}`;
+    expect(result[0].id).toBe(1);
+    expect(result[0].name).toBe("John");
+    expect(result[0].age).toBe(19);
+    expect(result[1].id).toBe(2);
+    expect(result[1].name).toBe("Jane");
+    expect(result[1].age).toBe(19);
+  });
+
+  test("update helper with undefined values and no columns", async () => {
```

Comment:
It is not immediately obvious in these tests what exactly the `"update helper"` is. Would add some docs to describe that.